core/Settings/Storage/Backend/PluginSettingsTable.php

```diff
@@ -48,6 +53,7 @@ public function __construct($pluginName, $userLogin)
 
         $this->pluginName = $pluginName;
         $this->userLogin = $userLogin;
+        $this->lockWrapper = new Lock(new LockBackend\MySqlLockBackend(), 'PluginSettingsTable');
```

Comment:
@katebutler as mentioned in chat we should ideally use DI here like : `StaticContainer::getContainer()->make(Lock::class, array('lockKeyStart' => 'myPrefix'));`